src/Thinktecture.Blazor.ViewTransitions/wwwroot/Thinktecture.Blazor.ViewTransitions.js

```diff
@@ -0,0 +1,12 @@
+export function startViewTransition(component, methodName) {
+    if (!document.startViewTransition) {
```

Comment:
This is a behavior change compared to the other packages: There, we will try to call the API even if it does not exist. Here, we implement our own fallback. I think it would be fine to skip the if branch completely.